.eslintrc.js

```diff
@@ -34,6 +34,87 @@ module.exports = {
     'ember/no-empty-attrs': 2,
     'ember/no-jquery': 2,
     'ember/no-observers': 2,
+
+    'ember/order-in-models': [
+      2,
+      {
+        order: ['attribute', 'relationship', 'single-line-function', 'multi-line-function'],
+      },
+    ],
+
+    'ember/order-in-components': [
+      2,
+      {
+        order: [
+          'service',
+          'property',
+          'empty-method',
+          'single-line-function',
+          'multi-line-function',
+          'observer',
+          'init',
+          [
+            // all the members of this array will be treated equally
+            'didReceiveAttrs',
+            'willRender',
+            'willInsertElement',
+            'didInsertElement',
+            'didRender',
+            'didUpdateAttrs',
+            'willUpdate',
+            'didUpdate',
+          ],
+          'method',
+          'actions',
+          ['willDestroyElement', 'willClearRender', 'didDestroyElement'], // all the members of this array will be treated equally
+        ],
+      },
+    ],
+
+    'ember/order-in-controllers': [
+      2,
+      {
+        order: [
+          'query-params',
+          'controller',
+          'service',
```

Comment:
Mostly curious, any particular reason for having services after controllers? My gut feel would have these the other way around.

Reply:
the controllers here refer to old school injections of controllers in others. https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/order-in-controllers.md.

At least on manage we don't inject it.

Reply:
Yeah I figured it was controller injection... for some reason it feels to me like services should come first though, maybe because they're more "global" than controllers? 🤷‍♂️

Reply:
Sure 😄